### ISSUE TITLE
I want to use "max_embeddings_multiples" argument to increase over 75 tokens.

### ISSUE THREAD

**Issue by MetisDeveloper** [huggingface/diffusers#1641]
I want to use "max_embeddings_multiples" argument to increase over 75 tokens.

I have this code.
```
from diffusers import DiffusionPipeline, AutoencoderKL
import torch
import random
from IPython.display import Image

model_id = "./anything-v3.0"
vae = AutoencoderKL.from_pretrained("./anything_vae")
pipe = DiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float32, vae=vae)
pipe.enable_attention_slicing()
pipe = pipe.to("cuda")
vae = pipe.to("cuda")
prompt = "a girl"
image = pipe(prompt, height=512, width=768, guidance_scale=9.5, num_inference_steps=50).images[0]
image.save("./anything_output.png")
```
<br>
I thought this code would work, but it didn't. (excerpt)

```
image = pipe(prompt, height=512, width=768, guidance_scale=9.5, num_inference_steps=50, max_embeddings_multiples=3).images[0]
```
<br>
<br>
Thank you in advance.

**Comment by SkyTNT** [comment]
See https://github.com/huggingface/diffusers/tree/main/examples/community#long-prompt-weighting-stable-diffusion

**Comment by MetisDeveloper** [comment]
Thank you for your renponse.
I tried that before and it didn't work so I thought I was doing it wrong.

Sure, this makes the weights and tokens valid, but doesn't it destroy the image?
An image normally drawn in 28 steps requires 50 steps.

This is the code.
```
from diffusers import DiffusionPipeline, AutoencoderKL
import torch
import random
from IPython.display import Image

model_id = "./anything-v3.0"
vae = AutoencoderKL.from_pretrained("./anything_vae")
pipe = DiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float32, vae=vae, custom_pipeline="lpw_stable_diffusion")
pipe.enable_attention_slicing()
pipe = pipe.to("cuda")
vae = pipe.to("cuda")
prompt = "a girl"
image = pipe(prompt, height=512, width=768, guidance_scale=9.5,num_inference_steps=28, max_embeddings_multiples=3).images[0]
image.save("./anything_output.png")
```

**Comment by SkyTNT** [comment]
Maybe you are using an inappropriate scheduler.
28 steps is good for EulerAncestralDiscreteScheduler but not for other schedulers such as PNDMScheduler

**Comment by MetisDeveloper** [comment]
I see, the scheduler was not good.
It was a rookie mistake.

Thank you very much for resolving this.
